ssl.go

```diff
@@ -1,195 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"io/ioutil"
-	"os"
-	"strings"
-	"time"
-
-	"github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
-)
-
-var cmdSSL = &Command{
```

Comment:
Empire will return an error message if they try to use the ssl-* commands with an old emp client. Do you think we should keep the ssl-* commands in here?